fuse_examples/classification/knight/eval/eval.py

```diff
@@ -235,10 +241,10 @@ def eval(task1_prediction_filename: str, task2_prediction_filename: str, target_
     """
     if len(sys.argv) == 1:
         # no arguments - set arguments inline - see details in function eval()
-        target_filename = "example/example_targets.csv"
-        task1_prediction_filename = "example/example_task1_predictions.csv"
-        task2_prediction_filename = "example/example_task2_predictions.csv"
-        output_dir = "example/results"
+        target_filename = "example2/example_targets.csv"
```

Comment:
leave it as the default or change to baseline prediction files

Reply:
my mistake, fixed. I suggest we sometime switch to handling inputs outside of version control to prevent such things.